consensus/polybft/proto/transport.pb.validate.go

```diff
@@ -39,18 +39,18 @@ var (
 // in the proto definition for this message. If any rules are violated, the
 // first error encountered is returned, or nil if there are no violations.
 func (m *TransportMessage) Validate() error {
-	return m.validate(false)
+	return m.validate()
```

Comment:
This code is auto-generated, so you should probably revert these changes, in case they aren't made when you re-run the protobuf compiler.